### DISCUSSION TITLE
Post model verbose names and the listdisplay fields in the admin.

### DISCUSSION THREAD

**Discussion by HelloMukama** [Radi85/Comment#197] [Q&A] [answered]
I just realised that for my `**Post**` model, the field verbose names override the listdisplay fields in the admin interface.
Now instead of say for a field `body`,
```
class Post(models.Model):
    body = models.TextField(verbose_name='Enter comment body below', max_length=100, blank=False, null=False
```)

Instead of using the title of a column in the admin as `body`,
it will use `Enter comment body below` for the column head

**Comment by abhiabhi94** [comment, marked as answer]
you can safely delete the `test` directory that contains these models, urls, etc and go on your merry way. There are just for illustrative and testing purpose.
